### ISSUE TITLE
Trigger custom action after submit

### ISSUE THREAD

**Issue by huvber** [marmelab/ng-admin#777]
I need to trigger some custom action after submit (for example return to the previous page, instead of remain in the same page). Is it possible to add a behaviour to the submit trigger?

**Comment by fzaninotto** [comment]
That's not built in, it's an enhancement request.

**Comment by huvber** [comment]
Thanks!

**Comment by kinnrot** [comment]
+1

**Comment by jamespsterling** [comment]
+1 for sure, some sort of custom callback would be awesome.

**Comment by fedotxxl** [comment]
+1. Please implement it

**Comment by wadjeroudi** [comment]
+1
Is there a workaround right now if you want to do something after entity changes ?

**Comment by lucval** [comment]
Yes implement your own logic in the form controller (https://github.com/marmelab/ng-admin/blob/master/src/javascripts/ng-admin/Crud/form/FormController.js).

It is as simple as a $state.go function call.

**Comment by devguf** [comment]
+1

**Comment by tarantsoff** [comment]
+1
